bridge/server/utils/configuration.ts

```diff
@@ -306,6 +308,7 @@ export function envToConfiguration(env: { [key in EnvVar]?: string }): BridgeOpt
     mode: env.NODE_ENV,
     logging: {
       enabledComponents: env.LOGGING_COMPONENTS,
+      defaultLogLevel: Level[env.LOG_LEVEL?.toLowerCase() as keyof typeof Level] ?? undefined,
```

Comment:
You should actually use `toUpperCase()`, since it uses the `key` and not the `value`.

```suggestion
      defaultLogLevel: Level[env.LOG_LEVEL?.toUpperCase() as keyof typeof Level] ?? undefined,
```